File: Data Structures & Algorithms/max-consecutive-ones/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMaxConsecutiveOnes(nums) {
        let counter = 0

        let temp = 0

        for(let val of nums){
            if(val == 1){
                temp++
                counter = Math.max(counter, temp)
            }else{
                temp = 0
            }
        }
        return counter
    }
}

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.py
class Solution:
    def findMin(self, nums: List[int]) -> int:
        l = 0
        r = len(nums)-1

        while l < r:
            middle = math.floor((l + r) /2)
            print(middle)
            if nums[middle] > nums[r]:
                l = middle + 1
            else:
                r = middle 
        
        return nums[r]

